components/notebook-dockerfiles/pytorch/requirements.txt

```diff
@@ -14,7 +14,7 @@ ipywidgets==7.7.1
 # Kubeflow Related Packages
 kfp==1.8.13
 kfp-server-api==1.8.3
-kfserving==0.6.1
+kserve==0.9.0
```

Comment:
Also tried with kserve 0.8.0.2

Reply:
so do we have both images?
Kubeflow 1.6 has KServe 0.8
https://github.com/kubeflow/manifests/tree/v1.6-branch#kubeflow-components-versions
release notes doesnt callout any breaking changes so should be ok

Reply:
0.8.0 needs the awscli and boto3 to be even older hence kept it at 0.9.0 for kserve